content/cli/kes-key/dek.md

```diff
@@ -8,51 +8,51 @@ tableOfContents: true
 
 ## Overview
 
-Generate a new data encryption key.
+Generate a new data encryption key from a Secret key on the KES server.
 
-The output of the command includes both a plaintext key and a ciphertext.
+The output of the command includes both a plaintext key and a ciphertext representation.
 The output resembles the following:
 
 ```
 plaintext:  kk/+NxO1LHb9ilbai7B9qo60649zNPmSVuJ2akEJFQ4=
 ciphertext: lbFBRVMyNTYtR0NNX1NIQTI1NtkgMTRlYjE3YWVjMTBjZDMxYTZiYzAwNmJhODFkNjM1ODnEEKOclQFBMYNZ3dVJPCrldAHEDLkZD9YgLpFW77+8b8Qw7Tn/6tFhyYUoFzS4+jYv8ty/Y5bqKzU6lPUEq/O8xEnYs92wEyvdSfTpTDEH8a8Q
 ```
 
+To encrypt or decrypt the keys, use `kes key encrypt` or `kes key decrypt`.
+
+{{< admonition type="caution">}}
+Avoid storing the plaintext on disk, as it allows decryption of data without requiring access to the Secret key used to generate the DEK.
+{{< /admonition>}}
+
 ## Syntax
 
 ```sh
 key key dek
-        [--insecure, -k]        \
-        [--enclave, -e <name>]  \
         <name>                  \
-        [<context>]
+        [<context>]             \
+        [--enclave, -e <name>]  \
+        [--insecure, -k]
 ```
 
 ## Parameters
 
-### `--insecure, -k`
-
-_Optional_
+### `name`
 
-Use this during testing and in non-production environments to bypass the TLS validation.
+{{< include "includes/params/name.md" >}}
 
-### `--enclave, -e`
+### `context`
 
 _Optional_
 
-The short name of the KES enclave to output information about.
-
-### `name`
+The context value to scope the request for a data encryption key.
```

Comment:
Just from reading this, I'm not sure what value one might use for this. Is there something useful to add about context values? (I also don't know much about KES so YMMV.)